package.json

```diff
@@ -15,7 +15,7 @@
     "watch": "cross-env TZ=UTC NODE_ICU_DATA=node_modules/full-icu jest -c jest.config.js --watch",
     "watch:integration": "cross-env TZ=UTC NODE_ICU_DATA=node_modules/full-icu jest -c jest.config.integration.js --watch",
     "lint:types": "tsc",
-    "lint:eslint": "eslint packages",
+    "lint:eslint": "eslint './packages/**/*.{ts,tsx,js,jsx}'",
```

Comment:
`eslint packages` by default would match only "*.js" files

Reply:
By the way, eslint has almost no rules. It has few plugins added, but hey have no sense for typescript projects. 
I'm going to setup eslint from scratch in the near future.